### DISCUSSION TITLE
Using single Database

### DISCUSSION THREAD

**Discussion by vishalinfyom** [spatie/laravel-multitenancy#269] [Q&A] [answered]
Hii,

Thanks for this great package first.

I want to create a basic demo using a single DB, using this package.

Let's say there are users but tenants are multiple and each tenant has different users's, so how i can create this sample example?

Thanks in advance.

**Comment by masterix21** [comment, marked as answer]
What about a column `tenant_id` in the users table?

**Reply by vishalinfyom** [reply]
Thanks @masterix21 

we have to add it manually right ?? and need to add tenant trait into that `User` model ?

**Reply by masterix21** [reply]
Yes
